formats/hocon/src/main/kotlin/kotlinx/serialization/hocon/HoconExceptions.kt

```diff
@@ -20,3 +20,6 @@ internal fun InvalidKeyKindException(value: ConfigValue) = SerializationExceptio
     "Value of type '${value.valueType()}' can't be used in HOCON as a key in the map. " +
             "It should have either primitive or enum kind."
 )
+
+internal fun UnsupportedFormatException(serializerName: String) =
```

Comment:
You can add `throw` to the function itself for more convenience, just like with e.g. `throwInvalidFloatingPointValueDecoded`: https://github.com/Kotlin/kotlinx.serialization/blob/60c632c970d7da02151fd7c920f40ebc269c0f29/formats/json/commonMain/src/kotlinx/serialization/json/internal/JsonExceptions.kt#L43

Reply:
Fixed